C-git-commands.asc

```diff
@@ -29,22 +29,22 @@ Bei diesem Befehl werden mehrere Dateien gelesen und beschrieben, so dass Sie We
 
 Der `git config` Befehl wird in fast jedem Kapitel des Buches benutzt.
 
-In <<ch01-getting-started#_first_time>> haben wir ihn verwendet, um unseren Namen, unsere E-Mail-Adresse und unsere Editor-Einstellung zu bestimmen, bevor wir mit Git anfangen konnten.
+In <<ch01-getting-started#,Git Basis-Konfiguration>> haben wir ihn verwendet, um unseren Namen, unsere E-Mail-Adresse und unsere Editor-Einstellung zu bestimmen, bevor wir mit Git anfangen konnten.
```

Comment:
Warum ist hier nicht, wie in den folgenden Absätzen, der englische Link enthalten? Funktioniert damit der Link? Müsste es so aussehen?
```suggestion
In <<ch01-getting-started#_first_time,Git Basis-Konfiguration>> haben wir ihn verwendet, um unseren Namen, unsere E-Mail-Adresse und unsere Editor-Einstellung zu bestimmen, bevor wir mit Git anfangen konnten.
```

Reply:
Das hat mich anfangs auch verwirrt. Ich hatte zuerst, wie du hier, die dt. Übersetzung immer eingefügt, weil im Preview der Link schräg aussah.
Als ich die "Korrektur" mal vergessen hatte, habe ich festgestellt, dass die Übersetzung der Links funktioniert (sie werden offensichtlich aus der verlinkten dt. Datei übernommen), sobald die Daten "gemergt"  und über den Build-Prozess zum Git-Server übertragen waren.
Ich habe mir deshalb angewöhnt, auf der Website der dt. Übersetzung, den Text abschließend nochmal zu prüfen. Wenn dann der Satz holprig klang, habe ich nochmal eingegriffen.

Sobald der Link eine bis dahin nicht übersetzte Datei betrifft, habe ich dort den betreffenden Link-Text gleich mit übersetzt. Bei dem derzeitigen Fortschritt der Gesamt-Übersetzung ist das, glaube ich, nicht mehr notwendig.

Reply:
```suggestion
In <<ch01-getting-started#_first_time>> haben wir ihn verwendet, um unseren Namen, unsere E-Mail-Adresse und unsere Editor-Einstellung zu bestimmen, bevor wir mit Git anfangen konnten.
```